### PR TITLE
Update the create snapshot repository documentation

### LINKED ISSUE #3028: [DOC] Inaccurate example(s) for register snapshots
**What do you want to do?**
 
- [ ] Request a change to existing documentation
- [ ] Add new documentation
- [X] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
Seems to affect all versions (only bothering to look at 1.3 and newer):
e.g.:
- https://opensearch.org/docs/1.3/api-reference/snapshots/create-repository/
- https://opensearch.org/docs/2.5/api-reference/snapshots/create-repository/

I don't think we're defining the `base_path` parameter correctly, or if we are then we aren't explaining it completely.

In the description of `base_path` the example (my/snapshot/directory) uses a relative path:

```
The path within the bucket where you want to store snapshots (e.g. my/snapshot/directory). Optional. If not specified, snapshots are stored in the bucket root.
```

Same with the example:

```
PUT _snapshot/my-opensearch-repo
{
  "type": "s3",
  "settings": {
    "bucket": "my-open-search-bucket",
    "base_path": "my/snapshot/directory"
  }
}
```

The documentation needs to explain how the path defined here translates to the mounted volume being used. Is it relative to the WKDIR defined in the Dockerfile? But that would only apply to Docker images. Does it use an environment variable like OPENSEARCH_HOME?

I think it probably *must* be an absolute path, but since this needs corrected anyway let's find out what happens when you specify a relative path and include that on the page as well.

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.

I don't have any additional info - if help is needed we'll have to rope in one of the devs.

### PR DESCRIPTION
**Pull request by ariamarble**
### Description

Updates and corrections to the create snapshot repository documentation.

### Issues Resolved

fixes #3028

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _api-reference/snapshots/create-repository.md [hunk 1]
```diff
@@ -53,18 +93,15 @@ Setting | Description
 `server_side_encryption` | Whether to encrypt snapshot files in the S3 bucket. This setting uses AES-256 with S3-managed keys. See [Protecting data using server-side encryption](https://docs.aws.amazon.com/AmazonS3/latest/dev/serv-side-encryption.html). Default is false. Optional.
 `storage_class` | Specifies the [S3 storage class](https://docs.aws.amazon.com/AmazonS3/latest/dev/storage-class-intro.html) for the snapshots files. Default is `standard`. Do not use the `glacier` and `deep_archive` storage classes. Optional.
 
-#### Example request
-
-The following requests register or update a repository called `my-first-repo`. Both require a request body as described in [Request fields](#request-fields).
-
-* `POST _snapshot/my-first-repo/` registers a snapshot repository.
+For the `base_path` parameter, you do not enter the `s3://` prefix when entering your s3 bucket details. Only the name of the bucket is required.
```
Comment (kolchfa-aws) [inline _api-reference/snapshots/create-repository.md:96]:
```suggestion
For the `base_path` parameter, do not enter the `s3://` prefix when entering your s3 bucket details. Only the name of the bucket is required.
```

File: _api-reference/snapshots/create-repository.md [hunk 2]
```diff
@@ -53,18 +93,15 @@ Setting | Description
...
+{: .note}
 
-* `PUT _snapshot/my-first-repo/` registers a snapshot repository if it does not exist; otherwise, updates information about the repository. 
+#### Example
```
Comment (kolchfa-aws) [inline _api-reference/snapshots/create-repository.md:99]:
```suggestion
#### Example request
```

File: _api-reference/snapshots/create-repository.md [hunk 3]
```diff
@@ -75,7 +112,7 @@ PUT _snapshot/my-opensearch-repo
 ```
 {% include copy-curl.html %}
 
-#### Example response
+#### Response
```
Comment (kolchfa-aws) [inline _api-reference/snapshots/create-repository.md:115]:
```suggestion
#### Example response
```

File: _tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md [hunk 4]
```diff
@@ -53,34 +53,17 @@ Before you can take a snapshot, you have to "register" a snapshot repository. A
...
-Request fields | Description
-:--- | :---
-`location` | The shared file system for snapshots. Required.
-`chunk_size` | Breaks large files into chunks during snapshot operations (e.g. `64mb`, `1gb`), which is important for cloud storage providers and far less important for shared file systems. Default is `null` (unlimited). Optional.
-`compress` | Whether to compress metadata files. This setting does not affect data files, which might already be compressed, depending on your index settings. Default is `false`. Optional.
-`max_restore_bytes_per_sec` | The maximum rate at which snapshots restore. Default is 40 MB per second (`40m`). Optional.
-`max_snapshot_bytes_per_sec` | The maximum rate at which snapshots take. Default is 40 MB per second (`40m`). Optional.
-`readonly` | Whether the repository is read-only. Useful when migrating from one cluster (`"readonly": false` when registering) to another cluster (`"readonly": true` when registering). Optional.
-
+You most likely will not need to specify anything but `location`. For allowed request parameters, see [Register or update snapshot repository API](https://opensearch.org/docs/latest/api-reference/snapshots/create-repository/)
```
Comment (kolchfa-aws) [inline _tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md:66]:
```suggestion
You most likely will not need to specify any parameters except `location`. For allowed request parameters, see [Register or update snapshot repository API](https://opensearch.org/docs/latest/api-reference/snapshots/create-repository/)
```

File: _tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md [hunk 5]
```diff
@@ -212,24 +196,9 @@ Request fields | Description
...
-`max_snapshot_bytes_per_sec` | The maximum rate at which snapshots take. Default is 40 MB per second (`40m`). Optional.
-`readonly` | Whether the repository is read-only. Useful when migrating from one cluster (`"readonly": false` when registering) to another cluster (`"readonly": true` when registering). Optional.
-`server_side_encryption` | Whether to encrypt snapshot files in the S3 bucket. This setting uses AES-256 with S3-managed keys. See [Protecting data using server-side encryption](https://docs.aws.amazon.com/AmazonS3/latest/dev/serv-side-encryption.html). Default is false. Optional.
-`storage_class` | Specifies the [S3 storage class](https://docs.aws.amazon.com/AmazonS3/latest/dev/storage-class-intro.html) for the snapshots files. Default is `standard`. Do not use the `glacier` and `deep_archive` storage classes. Optional.
-
+You most likely will not need to specify anything but `bucket` and `base_path`. For allowed request parameters, see [Register or update snapshot repository API](https://opensearch.org/docs/latest/api-reference/snapshots/create-repository/)
```
Comment (kolchfa-aws) [inline _tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md:201]:
```suggestion
You most likely will not need to specify any parameters except `bucket` and `base_path`. For allowed request parameters, see [Register or update snapshot repository API](https://opensearch.org/docs/latest/api-reference/snapshots/create-repository/)
```

File: _tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md [hunk 6]
```diff
@@ -241,22 +210,24 @@ You specify two pieces of information when you create a snapshot:
...
   "indices": "opensearch-dashboards*,my-index*,-my-index-2016",
   "ignore_unavailable": true,
   "include_global_state": false,
   "partial": false
 }
 ```
+{% include copy-curl.html %}
 
-Request fields | Description
+Request parameters | Description
```
Comment (kolchfa-aws) [inline _tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md:230]:
```suggestion
Request fields | Description
```

**Review by kolchfa-aws — APPROVED** [review]
LGTM! Left some comments.

**Review by cwillum — APPROVED** [review]
LGTM

File: _api-reference/snapshots/create-repository.md [hunk 7]
```diff
@@ -5,42 +5,65 @@ parent: Snapshot APIs
 nav_order: 1
 ---
 
-## Register or update snapshot repository
+# Register or update snapshot repository
 
-Registers a new repository to store snapshots, or updates information for an existing repository. Repositories can be of two types:
+You can register a new repository to store snapshots, or update information for an existing repository, using the snapshots API.
```
Comment (natebower) [inline _api-reference/snapshots/create-repository.md:10]:
```suggestion
You can register a new repository in which to store snapshots or update information for an existing repository by using the snapshots API.
```

File: _api-reference/snapshots/create-repository.md [hunk 8]
```diff
@@ -5,42 +5,65 @@ parent: Snapshot APIs
 nav_order: 1
 ---
 
-## Register or update snapshot repository
+# Register or update snapshot repository
```
Comment (natebower) [inline _api-reference/snapshots/create-repository.md:8]:
```suggestion
# Registering or updating a snapshot repository
```

File: _api-reference/snapshots/create-repository.md (hunk 7 above)
Comment (natebower) [reply _api-reference/snapshots/create-repository.md:10]: Should "Snapshots API" be capitalized?

File: _api-reference/snapshots/create-repository.md [hunk 9]
```diff
@@ -5,42 +5,65 @@ parent: Snapshot APIs
...
 
-Registers a new repository to store snapshots, or updates information for an existing repository. Repositories can be of two types:
+You can register a new repository to store snapshots, or update information for an existing repository, using the snapshots API.
 
-* File system (`fs`)
+Snapshot repositories can be of two types:
```
Comment (natebower) [inline _api-reference/snapshots/create-repository.md:12]:
```suggestion
There are two types of snapshot repositories:
```

File: _api-reference/snapshots/create-repository.md [hunk 10]
```diff
@@ -5,42 +5,65 @@ parent: Snapshot APIs
...
 
-* Amazon S3 bucket (`s3`)
+* File system (`fs`). For steps on creating an `fs` repository, see [Register repository shared file system]({{site.url}}{{site.baseurl}}/availability-and-recovery/snapshots/snapshot-restore/#shared-file-system).
```
Comment (natebower) [inline _api-reference/snapshots/create-repository.md:14]:
```suggestion
* File system (`fs`): For instructions on creating an `fs` repository, see [Register repository shared file system]({{site.url}}{{site.baseurl}}/availability-and-recovery/snapshots/snapshot-restore/#shared-file-system).
```

File: _api-reference/snapshots/create-repository.md [hunk 11]
```diff
@@ -5,42 +5,65 @@ parent: Snapshot APIs
...
+* Amazon S3 bucket (`s3`). For steps on creating an `s3` repository, see [Register repository Amazon S3]({{site.url}}{{site.baseurl}}/availability-and-recovery/snapshots/snapshot-restore/#amazon-s3).
```
Comment (natebower) [inline _api-reference/snapshots/create-repository.md:15]:
```suggestion
* Amazon Simple Storage Service (Amazon S3) bucket (`s3`): For instructions on creating an `s3` repository, see [Register repository Amazon S3]({{site.url}}{{site.baseurl}}/availability-and-recovery/snapshots/snapshot-restore/#amazon-s3).
```

File: _api-reference/snapshots/create-repository.md [hunk 12]
```diff
@@ -5,42 +5,65 @@ parent: Snapshot APIs
...
 
-To learn more about repositories, see [Register repository]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore#register-repository).
+For steps on creating a repository, see [Register repository]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore#register-repository).
```
Comment (natebower) [inline _api-reference/snapshots/create-repository.md:17]:
```suggestion
For instructions on creating a repository, see [Register repository]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore#register-repository).
```

File: _api-reference/snapshots/create-repository.md (hunk 8 above)
Comment (natebower) [reply _api-reference/snapshots/create-repository.md:8]: Topic titles: https://github.com/opensearch-project/documentation-website/blob/main/STYLE_GUIDE.md#topic-titles

File: _api-reference/snapshots/create-repository.md (hunk 11 above)
Comment (natebower) [reply _api-reference/snapshots/create-repository.md:15]: AWS service names should always be defined on first appearance.

File: _api-reference/snapshots/create-repository.md [hunk 13]
```diff
@@ -5,42 +5,65 @@ parent: Snapshot APIs
...
+POST /_snapshot/my-first-repo/ 
+PUT /_snapshot/my-first-repo/
+```
+
+## Path parameters
 
 Parameter | Data type | Description
 :--- | :--- | :---
-repository | String | Repostory name. |
+repository | String | Repository name. |
```
Comment (natebower) [inline _api-reference/snapshots/create-repository.md:30]:
```suggestion
repository | String | Repository name |
```

File: _api-reference/snapshots/create-repository.md [hunk 14]
```diff
@@ -5,42 +5,65 @@ parent: Snapshot APIs
...
 
-### Request fields
+## Request parameters
 
-Allowable parameters depend on the repository type.
+Request parameters depend on whether the repository is `fs` or `s3`.
```
Comment (natebower) [inline _api-reference/snapshots/create-repository.md:34]:
```suggestion
Request parameters depend on the type of repository: `fs` or `s3`.
```

File: _api-reference/snapshots/create-repository.md [hunk 15]
```diff
@@ -5,42 +5,65 @@ parent: Snapshot APIs
...
 
-#### fs repository type
+### fs repository
 
-Setting | Description
+Request field | Description
 :--- | :---
-`location` | The shared file system for snapshots. Required.
+`location` | The file system directory for snapshots, such as a mounted directory from a file server or a samba share. Must be accessible by all nodes. Required.
```
Comment (natebower) [inline _api-reference/snapshots/create-repository.md:40]:
```suggestion
`location` | The file system directory for snapshots, such as a mounted directory from a file server or a Samba share. Must be accessible by all nodes. Required.
```

File: _api-reference/snapshots/create-repository.md [hunk 16]
```diff
@@ -5,42 +5,65 @@ parent: Snapshot APIs
...
 `chunk_size` | Breaks large files into chunks during snapshot operations (e.g. `64mb`, `1gb`), which is important for cloud storage providers and far less important for shared file systems. Default is `null` (unlimited). Optional.
 `compress` | Whether to compress metadata files. This setting does not affect data files, which might already be compressed, depending on your index settings. Default is `false`. Optional.
 `max_restore_bytes_per_sec` | The maximum rate at which snapshots restore. Default is 40 MB per second (`40m`). Optional.
 `max_snapshot_bytes_per_sec` | The maximum rate at which snapshots take. Default is 40 MB per second (`40m`). Optional.
 `readonly` | Whether the repository is read-only. Useful when migrating from one cluster (`"readonly": false` when registering) to another cluster (`"readonly": true` when registering). Optional.
 
-#### s3 repository type
+#### Example request
+
+The following example registers an `fs` repository using the local directory  `/mnt/snapshots` as `location`.
```
Comment (natebower) [inline _api-reference/snapshots/create-repository.md:49]:
```suggestion
The following example registers an `fs` repository using the local directory `/mnt/snapshots` as `location`.
```

File: _api-reference/snapshots/create-repository.md [hunk 17]
```diff
@@ -5,42 +5,65 @@ parent: Snapshot APIs
...
+}
+```
+{% include copy-curl.html %}
+
+### s3 repository
+
+Request field | Description
 :--- | :---
-`base_path` | The path within the bucket where you want to store snapshots (e.g. `my/snapshot/directory`). Optional. If not specified, snapshots are stored in the bucket root.
+`base_path` | The path within the bucket where you want to store snapshots (for example, `my/snapshot/directory`). Optional. If not specified, snapshots are stored in the s3 bucket root.
```
Comment (natebower) [inline _api-reference/snapshots/create-repository.md:66]:
```suggestion
`base_path` | The path within the bucket in which you want to store snapshots (for example, `my/snapshot/directory`). Optional. If not specified, snapshots are stored in the S3 bucket root.
```

File: _api-reference/snapshots/create-repository.md [hunk 18]
```diff
@@ -53,18 +76,15 @@ Setting | Description
 `server_side_encryption` | Whether to encrypt snapshot files in the S3 bucket. This setting uses AES-256 with S3-managed keys. See [Protecting data using server-side encryption](https://docs.aws.amazon.com/AmazonS3/latest/dev/serv-side-encryption.html). Default is false. Optional.
 `storage_class` | Specifies the [S3 storage class](https://docs.aws.amazon.com/AmazonS3/latest/dev/storage-class-intro.html) for the snapshots files. Default is `standard`. Do not use the `glacier` and `deep_archive` storage classes. Optional.
 
-#### Example request
-
-The following requests register or update a repository called `my-first-repo`. Both require a request body as described in [Request fields](#request-fields).
+For the `base_path` parameter, do not enter the `s3://` prefix when entering your s3 bucket details. Only the name of the bucket is required.
```
Comment (natebower) [inline _api-reference/snapshots/create-repository.md:79]:
```suggestion
For the `base_path` parameter, do not enter the `s3://` prefix when entering your S3 bucket details. Only the name of the bucket is required.
```

File: _tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md [hunk 19]
```diff
@@ -53,34 +53,17 @@ Before you can take a snapshot, you have to "register" a snapshot repository. A
...
-:--- | :---
-`location` | The shared file system for snapshots. Required.
-`chunk_size` | Breaks large files into chunks during snapshot operations (e.g. `64mb`, `1gb`), which is important for cloud storage providers and far less important for shared file systems. Default is `null` (unlimited). Optional.
-`compress` | Whether to compress metadata files. This setting does not affect data files, which might already be compressed, depending on your index settings. Default is `false`. Optional.
-`max_restore_bytes_per_sec` | The maximum rate at which snapshots restore. Default is 40 MB per second (`40m`). Optional.
-`max_snapshot_bytes_per_sec` | The maximum rate at which snapshots take. Default is 40 MB per second (`40m`). Optional.
-`readonly` | Whether the repository is read-only. Useful when migrating from one cluster (`"readonly": false` when registering) to another cluster (`"readonly": true` when registering). Optional.
-
+You most likely will not need to specify any parameters except `location`. For allowed request parameters, see [Register or update snapshot repository API](https://opensearch.org/docs/latest/api-reference/snapshots/create-repository/)
```
Comment (natebower) [inline _tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md:66]:
```suggestion
You will most likely not need to specify any parameters except for `location`. For allowed request parameters, see [Register or update snapshot repository API](https://opensearch.org/docs/latest/api-reference/snapshots/create-repository/).
```

File: _tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md [hunk 20]
```diff
@@ -212,24 +196,9 @@ Request fields | Description
...
-`max_snapshot_bytes_per_sec` | The maximum rate at which snapshots take. Default is 40 MB per second (`40m`). Optional.
-`readonly` | Whether the repository is read-only. Useful when migrating from one cluster (`"readonly": false` when registering) to another cluster (`"readonly": true` when registering). Optional.
-`server_side_encryption` | Whether to encrypt snapshot files in the S3 bucket. This setting uses AES-256 with S3-managed keys. See [Protecting data using server-side encryption](https://docs.aws.amazon.com/AmazonS3/latest/dev/serv-side-encryption.html). Default is false. Optional.
-`storage_class` | Specifies the [S3 storage class](https://docs.aws.amazon.com/AmazonS3/latest/dev/storage-class-intro.html) for the snapshots files. Default is `standard`. Do not use the `glacier` and `deep_archive` storage classes. Optional.
-
+You most likely will not need to specify any parameters except `bucket` and `base_path`. For allowed request parameters, see [Register or update snapshot repository API](https://opensearch.org/docs/latest/api-reference/snapshots/create-repository/)
```
Comment (natebower) [inline _tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md:201]:
```suggestion
You will most likely not need to specify any parameters except for `bucket` and `base_path`. For allowed request parameters, see [Register or update snapshot repository API](https://opensearch.org/docs/latest/api-reference/snapshots/create-repository/).
```

**Review by natebower — COMMENTED** [review]
@ariamarble Please see my comments and changes and let me know if you have any questions. Thanks!

**Pull request merged**